### DISCUSSION TITLE
Night time person detection doesn't detect.

### DISCUSSION THREAD

**Discussion by tomhusband** [blakeblackshear/frigate#5833] [Ask A Question]
I've had my Amcrest camera (IP8M-2496EW) running for some time now but only recently noticed that nothing is picked up at night. During the day people are detected but at night there's nothing. I've gone into the camera settings and tweaked contrast, brightness and made sure the day/night profiles were on but it hasn't helped.

I've read about the improved_contrast: True setting and added it to frigate.yml but still no improvement. I guess placement of the code is critical because placed in some locations Frigate won't run. I have Home Assistant running in Proxmox and don't have a Google Coral as of yet.

Do I have the location correct in my frigate.yml (it's at the end)? Anything else I should look at?

Many thanks for any hints. 
https://pastebin.com/yXtGgBkA

**Comment by NickM-27** [comment]
Location looks fine to enable it for the driveway cam. Only to Way to know for sure why it's not detecting is to watch the debug live view with object and motion boxes enabled

**Reply by tomhusband** [reply]
OK thanks. So the position of `improved_contrast: True` is good?

I guess I'll have to get the wife to walk around in the drive way tonight. She's gonna love that. What would I be looking for in debug live view?
edit: In debug live view I have motion box but I don't see object box.

**Reply by NickM-27** [reply]
If you have motion box but no object box then it's either not detecting objects or they are being filtered out by your config. I'd suggest posting your entire config

**Reply by tomhusband** [reply]
It detects people in the daytime but not at night. I think all of frigate.yml is here: https://pastebin.com/yXtGgBkA

**Reply by NickM-27** [reply]
setting a threshold of 0.8 for person is probably part of the issue, will definitely get lower scores at night and probably not reach that threshold.

**Reply by NickM-27** [reply]
The default model has maximum possible scores of 0.84 so that is pushing it pretty close

**Reply by tomhusband** [reply]
Hmmm, that makes sense. Can I set up a threshold for daytime and one for nighttime?

**Reply by NickM-27** [reply]
currently no that is not supported, there is already a pinned feature request for this

**Reply by tomhusband** [reply]
OK, thanks again. I lowered the threshold to 0.7 so We'll see what happens with that. Should I go lower?

**Reply by NickM-27** [reply]
0.7 is the default and that should be fine.
